assets/src/edit-story/components/inspector/prepublish/prepublishCheckpointState.js

```diff
@@ -23,14 +23,17 @@ export const PPC_CHECKPOINT_STATE = {
 export const PPC_CHECKPOINT_ACTION = {
   ON_DIRTY_STORY: 'story is no longer empty',
   ON_PUBLISH_CLICKED: 'publish button is pressed',
-  ON_STORY_HAS_5_PAGES: 'story has minimal suggested amount of pages',
+  ON_STORY_HAS_2_PAGES: "story 'recommended' section enabled",
+  ON_STORY_HAS_5_PAGES: "story 'high priority' section enabled",
```

Comment:
Do y'all think we need these? It feels like redundant state that are cover with `STORY_EVENTS` 🤔 
![image](https://user-images.githubusercontent.com/22185279/115930647-13fe5380-a447-11eb-964c-9883453bdba2.png)

Reply:
I think it's whatever you want. We could also do `onPageAdded` and inspect the story in the listener too 🤷 .

> Do y'all think we need these? 

Do you mean the separation between 2 & 5 pages for events? or the story events for page change in general?